lib/src/driver.dart

```diff
@@ -324,11 +338,35 @@ class Driver {
     _context = context;
   }
 
+  void _processAnalysisOptions(CommandLineOptions options) {
+
+    // Find file
+    var filePath = options.analysisOptionsFile != null
+        ? options.analysisOptionsFile
+        : analysisOptionsFileName;
+    var file = new File(filePath);
+    if (!file.existsSync()) {
+      return;
+    }
+
+    List<OptionsProcessor> optionsProcessors =
+        AnalysisEngine.instance.optionsPlugin.optionsProcessors;
+
+    // Read file and notify processors
+    try {
+      String contents = file.readAsStringSync();
+      Map<String, YamlNode> options = new OptionsFileParser().parse(contents);
+      optionsProcessors
+          .forEach((OptionsProcessor p) => p.optionsProcessed(options));
+    } catch (e) {
+      optionsProcessors.forEach((OptionsProcessor p) => p.onError(e));
```

Comment:
The method 'onError' is typed to take an 'Exception', but 'e' can be anything. We should add a type to the catch clause.